scripts/extract_policies.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```

I usually add this to my scripts
- exit on error
- exit on usage of undefined variable
- set exit status based on whole pipeline, not just last command

Reply:
+ 1 here, we forgot to add it too to extract_images.sh.

Reply:
Given that now we have shell scripts, I think we should run shellcheck as part of workflows/ci.yaml.